src/Phuong.eShop.WebApp/components/catalog/catalog-item.tsx

```diff
@@ -19,8 +34,8 @@ const CatalogItem = (props: CatalogItemProps) => {
         </div>
       </CardContent>
       <CardFooter>
-        <Button className="w-full">
-          <ShoppingBag className="mr-2 h-4 w-4" /> Add to cart
+        <Button className="w-full" onClick={handleAdd}>
```

Comment:
Create `AddToCartButton` as a client component, then handle logic add cart inside that component